client/src/pages/Messages/Message/Message.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement } from 'react';
+import { MessageInterface } from '../../../interface/Conversation';
+import moment from 'moment';
+import { Grid, ListItemText, ListItem, Avatar, Typography, ListItemAvatar, List } from '@mui/material';
+import { useStyles } from './useStyles';
+import { useAuth } from '../../../context/useAuthContext';
+
+interface Props {
+  message: MessageInterface;
+}
+
+export default function Message({ message }: Props): ReactElement {
+  const { loggedInUser } = useAuth();
+
+  const sender = message.sender.name === loggedInUser?.name ? true : false;
```

Comment:
Maybe we should compare the logged in user's id with the participants. Right now we are comparing names, what if the logged in user changes his name from the Edit Profile form?